### ISSUE TITLE
Update 2.4 stuck at 0%

### ISSUE THREAD

**Issue by xme** [CIRCL/AIL-framework#424]
I also noticed that the update process is stuck at 0%:

```
Backgroud Update: v2.4                0/1
Updating: domain update ...           %0
```
Any way to check the status of the upgrade process?

**Comment by Terrtia** [comment]
Hey @xme !
Sorry for the late reply,
This error is related to #423

You need to launch the background update manually: (with AIL running)
```bash
. ./AILENV/bin/activate
cd update/v2.4
./Update_domain.py
```

Launch this script to reset the update progress (UI)

```bash
. ./AILENV/bin/activate
cd bin
./update-background.py
```

**Comment by xme** [comment]
NP, no hurry. Hmmm... This is what I have:

```
(AILENV) root@ail:/opt/AIL-framework/update/v2.4# ./Update_domain.py
--tt3j2x45ulyaarzc.onion
Traceback (most recent call last):
  File "./Update_domain.py", line 71, in <module>
    update_domain_by_item(domain, item_dict['id'])
  File "./Update_domain.py", line 36, in update_domain_by_item
    item_correlation = Item.get_item_all_correlation(item_id)
  File "/opt/AIL-framework/bin/packages/Item.py", line 174, in get_item_all_correlation
    correlation_names = Correlation.get_all_correlation_names()
AttributeError: module 'Correlation' has no attribute 'get_all_correlation_names'
```

**Comment by Terrtia** [comment]
Wrong import ...

Should be fixed with 9d652525c48855d596f959b979b5c61393fb8e40 and d2e59c935a976ca30f3d1c98f7af89bb3b6a4b81

**Comment by xme** [comment]
Confirmed working! Tx!
